### DISCUSSION TITLE
能不能给简化首页风格-清爽加个限宽？

### DISCUSSION THREAD

**Discussion by ZiSaAcZ7** [the1812/Bilibili-Evolved#2444] [功能建议 / Ideas]
如图，我发现不知道从什么时候开始清爽模式的首页就是整个页面宽度了（我记得以前是固定一个宽度），下面分区推荐的地方有好大的空很难受![image](https://user-images.githubusercontent.com/57062426/139560039-0331a855-8685-45b1-bcc8-6adedd4a7d35.png)

**Comment by the1812** [comment]
之前换了个限宽, 你分辨率是多少

**Reply by ZiSaAcZ7** [reply]
3840*2160，win缩放150%，浏览器缩放110%。

**Reply by ZiSaAcZ7** [reply]
是不是可以加个自定义？
